Jenkinsfile.github

```diff
@@ -80,19 +80,21 @@ pipeline {
 
             steps {
                 echo "Docker args are ${env.DOCKER_ARGS}"
-                sh "make image"
+                withCredentials([usernamePassword(credentialsId: 'artifactory-algol60-publish', passwordVariable: 'ARTIFACTORY_PASSWORD', usernameVariable: 'ARTIFACTORY_USERNAME')]) {
+                    sh "echo \$ARTIFACTORY_PASSWORD | docker login artifactory.algol60.net --username \$ARTIFACTORY_USERNAME --password-stdin"
```

Comment:
Need to authenticate to docker registry before running `make image`, because it performs push now.